source/access.tex

```diff
@@ -116,8 +116,9 @@
 the class appearing outside the class's \grammarterm{member-specification}{.}
 \begin{note} This access also applies to implicit references to constructors,
 conversion functions, and destructors. \end{note}
-\begin{example}
 
+\pnum
+\begin{example}
```

Comment:
Yeah, the star doesn't work well for interior changes :-S

OK, good, if this is all analyzed case-by-case, then I'm happy with it.